Modules/Campaign/Policies/CampaignPolicy.php

```diff
@@ -21,33 +21,46 @@ public function __construct()
     }
 
     /**
-     * Determine whether the user can view the post.
+     * Determine whether the user can view any campaigns.
      *
-     * @param \Modules\User\Entities\User  $user
-     * @param \Modules\Campaign\Entities\Campaign  $campaign
+     * @param \Modules\User\Entities\User $user
+     * @return mixed
+     */
+    public function viewAny(User $user)
+    {
+        return $user->role === 'brand';
+    }
+
+    /**
+     * Determine whether the user can view the campaign.
+     *
+     * @param \Modules\User\Entities\User $user
+     * @param \Modules\Campaign\Entities\Campaign $campaign
      * @return mixed
      */
     public function view(User $user, Campaign $campaign)
     {
-        return TRUE;
+        return $user->id === $campaign->user_id;
```

Comment:
in order to not repeat ourselves it's better to create "protected function access" store this logic there and call access function in view, edit, delete etc methods.